inbox/pubsub-server-info.xml

```diff
@@ -0,0 +1,275 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE xep SYSTEM 'xep.dtd' [
+  <!ENTITY % ents SYSTEM '../xep.ent'>
+  %ents;
+  ]>
+<?xml-stylesheet type='text/xsl' href='xep.xsl'?>
+<xep xmlns="">
+<header>
+  <title>PubSub Server Information</title>
+  <abstract>This document defines a data format whereby basic information of an XMPP domain can be expressed and exposed over pub-sub.</abstract>
+  &LEGALNOTICE;
+  <number>xxxx</number>
+  <status>ProtoXEP</status>
+  <type>Standards Track</type>
+  <sig>Standards</sig>
+  <approver>Council</approver>
+  <dependencies/>
+  <supersedes/>
+  <supersededby/>
+  <shortname>serverinfo</shortname>
+  <author>
+    <firstname>Guus</firstname>
+    <surname>der Kinderen</surname>
+    <email>guus.der.kinderen@gmail.com</email>
+    <jid>guus.der.kinderen@igniterealtime.org</jid>
+  </author>
+  <revision>
+    <version>1.0.0</version>
+    <date>2023-12-19</date>
+    <initials>gdk</initials>
+    <remark>
+      <ul>
+        <li>Initial version.</li>
+      </ul>
+    </remark>
+  </revision>
+</header>
+<section1 topic="Introduction" anchor="intro">
+  <p>To facilitate discovery of information of individual domains in an XMPP-based network, this specification defines a data format to define basic information for individual XMPP domains. By leveraging &xep0060; this information can efficiently be shared with applications that compose an overview of the larger XMPP network.</p>
+</section1>
+  <section1 topic="Requirements" anchor="requirements">
+    <ul>
+      <li>Describe links between nodes in an XMPP-based network, by enumerating connections used for federation between XMPP domains.</li>
+      <li>An extensible data format, allowing additional data (such as that defined in &xep0092;) to be retrievable without requiring additional round-trips.</li>
+    </ul>
+  </section1>
+<section1 topic="Discovering Support" anchor="disco">
+  <p>Domains supporting the publication of Server Information data, as described in this document, MUST advertise the fact by announcing a &xep0030; feature of 'urn:xmpp:serverinfo:0'. This signifies that an administrative entity approved the publication of data, which is important for the opt-in mechanism described in <link url="#privacy">Privacy Considerations section</link> of this document.</p>
+  <p>The pub-sub service address and node in which Server Information data is advertised SHOULD be specified using a &xep0128;. These pub-sub coordinates MUST be scoped using a FORM_TYPE of "http://jabber.org/network/serverinfo" (as already specified in XEP-0128) and data form fields registered for this purpose as defined in the <link url="#registrar">XMPP Registrar Considerations section</link> of this document.</p>
```

Comment:
XEP-0157 specifies the 'serverinfo' form, XEP-0128 just uses it as example. Hence:

```suggestion
  <p>The pub-sub service address and node in which Server Information data is advertised SHOULD be specified using a &xep0128;. These pub-sub coordinates MUST be scoped using a FORM_TYPE of "http://jabber.org/network/serverinfo" (as already specified in &xep0157;) and data form fields registered for this purpose as defined in the <link url="#registrar">XMPP Registrar Considerations section</link> of this document.</p>
```